vault/store.go

```diff
@@ -10,6 +10,7 @@ import (
 	"runtime"
 	"sync"
 
+	simplejson "github.com/bitly/go-simplejson"
 	. "github.com/nknorg/nkn/common"
 )
 
```

Comment:
Shall we change WalletStoreVersion from 0.0.1 to something like 0.0.2 or 0.1?

Reply:
Yes, it needs to be changed synchronously to SDK.

Reply:
We can make the change now and people will make changes on the SDK side. It doesn't need to be synchronized because the version is not used anywhere AFAIK.

Reply:
I had added in vault/wallet.go.